00-basics/003-tests/Udemi/OOP/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+<script src="script.js"></script>
```

Comment:
since there is no actual layout here, probably you can launch script.js directly from WebStorm; you'll also get full debugging support that way